hosted_control_planes/hcp-authentication-authorization.adoc

```diff
@@ -9,9 +9,28 @@ toc::[]
 The {product-title} control plane includes a built-in OAuth server. You can obtain OAuth access tokens to authenticate to the {product-title} API. After you create your hosted cluster, you can configure OAuth by specifying an identity provider.
 
 include::modules/hcp-configuring-oauth.adoc[leveloffset=+1]
+
 include::modules/hcp-configuring-oauth-console.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * To know more about supported identity providers, see xref:../authentication/understanding-identity-provider.adoc#understanding-identity-provider["Understanding identity provider configuration"] in _Authentication and authorization_.
+
+[id="hcp-cco-aws-sts_{context}"]
+== Using the CCO in a hosted cluster on {aws-short}
```

Comment:
```suggestion
== Assigning components IAM roles by using the CCO in a hosted cluster on {aws-short}
```
Just a thought. Is the goal of this task to use the CCO in a hosted cluster, or is it to assign components IAM roles by using the CCO in a hosted cluster?

Reply:
As per the text `Assigning components IAM...` fits better. Addressed the suggestion.